twiggy/tests/all/main.rs

```diff
@@ -53,6 +53,7 @@ macro_rules! test {
 
             let actual = String::from_utf8_lossy(&output.stdout);
 
+            // @@@ unsure about the test changes I introduced in this patch. =S
```

Comment:
The only one I'm not sure about is `wee_alloc_csv` test. The other ones seem fine.
I'm just making a note here to double check those numbers are as expected.
I think the comment can be safely deleted.